<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="prefixLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prefixField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="attributesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attributes to include in file name:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="exposureButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure time</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="lightButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LIGHT/DARK</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="binningButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Binning mode</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="temperatureButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CCD Temperature</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="dateButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date and Time</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Download creates a file name for each file, based on a prefix and some of the attributes of the file. If no attributes are selected, the task id is used instead.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DownloadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>exposureToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>binningButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>binningToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>shutterToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>filterToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>temperatureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>temperatureToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>dateToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prefixField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DownloadDialog</receiver>
   <slot>textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exposureToggled(bool)</slot>
  <slot>binningToggled(bool)</slot>
  <slot>shutterToggled(bool)</slot>
  <slot>filterToggled(bool)</slot>
  <slot>temperatureToggled(bool)</slot>
  <slot>dateToggled(bool)</slot>
  <slot>textChanged(QString)</slot>
 </slots>
</ui>
